src/parser.act

```diff
@@ -467,6 +467,40 @@
 		ast->max_properties = c;
 	@};
 
+	<set-min-length>: (n :number) -> () = @{
+                ast_count c;
+
+		if (!(@n > 0)) {
+			xerr(lex_state, KW_MIN_PROPERTIES, "expected > 0, got %f", @n);
+		}
```

Comment:
the documentation for `minLength` says:
> Omitting this keyword has the same behavior as a value of 0.

so I figure 0 is a valid value here. What do you think?

Reply:
Yes, you're right!

Reply:
also need to fix the KW_MIN_PROPERTIES to be KW_MIN_LENGTH